skimage/io/_plugins/matplotlib_plugin.py

```diff
@@ -169,10 +169,33 @@ def imshow_collection(ic, *args, **kwargs):
     """Display all images in the collection.
 
     """
-    fig, axes = plt.subplots(1, len(ic))
+    N = len(ic)
+    print N
```

Comment:
This is a SyntaxError in Python 3. You should consider upgrading. ;) At any rate, please don't put print statements in final code.